dask_sql/physical/utils/ml_classes.py

```diff
@@ -0,0 +1,130 @@
+def get_cpu_classes():
+    try:
+        from sklearn.utils import all_estimators
+
+        cpu_classes = {
+            k: v.__module__ + "." + v.__qualname__ for k, v in all_estimators()
+        }
+    except ImportError:
+        cpu_classes = {}
+
+    # Boosting libraries
+    cpu_classes["LGBMModel"] = "lightgbm.LGBMModel"
+    cpu_classes["LGBMClassifier"] = "lightgbm.LGBMClassifier"
+    cpu_classes["LGBMRegressor"] = "lightgbm.LGBMRegressor"
+    cpu_classes["LGBMRanker"] = "lightgbm.LGBMRanker"
+    cpu_classes["XGBRegressor"] = "xgboost.XGBRegressor"
+    cpu_classes["XGBClassifier"] = "xgboost.XGBClassifier"
+    cpu_classes["XGBRanker"] = "xgboost.XGBRanker"
+    cpu_classes["XGBRFRegressor"] = "xgboost.XGBRFRegressor"
+    cpu_classes["XGBRFClassifier"] = "xgboost.XGBRFClassifier"
+    cpu_classes["DaskXGBClassifier"] = "xgboost.dask.DaskXGBClassifier"
+    cpu_classes["DaskXGBRegressor"] = "xgboost.dask.DaskXGBRegressor"
+    cpu_classes["DaskXGBRanker"] = "xgboost.dask.DaskXGBRanker"
+    cpu_classes["DaskXGBRFRegressor"] = "xgboost.dask.DaskXGBRFRegressor"
+    cpu_classes["DaskXGBRFClassifier"] = "xgboost.dask.DaskXGBRFClassifier"
```

Comment:
Just noticed that these classes are shared between both the CPU and GPU classes; knowing this, would it make sense to make this a common dict that's added to both mappings with `dict.update` to cut down on repetition?

Reply:
Done, thanks!